src/hooks.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react'
+import { ContainerRef, RefComponent } from 'react-native-collapsible-tab-view'
```

Comment:
Isn't keep relative imports better? `./types` To avoid any circular import. If I'm missing something let me know.

Reply:
Indeed. Looks like the auto import fix messed this up.

However. I'd prefer #93 to this.